server/db.js

```diff
@@ -112,35 +111,53 @@ function getSiblingSizes(shas, chunk) {
 		.andWhere('cg.chunk', chunk);
 }
 
-exports.getChunkGroupChartData = async (period, chunk, loadedChunks, branch = 'master') => {
-	const lastCount = periodToLastCount(period);
+exports.getChunkGroupChartData = async (period, chunks, loadedChunks, branch = 'master') => {
+	chunks = _.split(chunks, ',');
+	loadedChunks = _.split(loadedChunks, ',');
```

Comment:
If `loadedChunks` is null or undefined, the `split` call will produce `[ '' ]`. It's unlikely we want that.